cypress/support/checkout.js

```diff
@@ -10,12 +10,12 @@ Cypress.Commands.add('checkOutItem', (userBarcode, itemBarcode) => {
   cy.do([
     TextField('Patron identifier').fillIn(userBarcode),
     Pane('Scan patron card').find(Button('Enter')).click(),
-
-    Button(userBarcode).exists(),
-
+  ]);
+  // wait form to load
+  cy.wait(1000);
```

Comment:
based on comment - good point to try to add waiter of special form loading. If it's more difficult - describe this difficulty in comment.